recipe/meta.yaml

```
@@ -79,15 +82,14 @@ test:
   commands:
     - test -f $PREFIX/lib/libcamera.so
     - test -f $PREFIX/include/libcamera/libcamera/libcamera.h
+    - cam -l   # [variant == "upstream"]
```

This is one way to skip the test test for `rpi` 
The suggestion provides an other.


```suggestion
    - cam -l   # [variant != "rpi"]
```

But the real question is WHY did rpi team remove it. 

I think for consistency of packaging, you should either have it in both, or neither in both.